src/index.js

```diff
@@ -52,7 +56,11 @@ export default ({ types: t }) => {
         const lastNode = body[lastIndex]
 
         // skip unreturnable statements
-        if (unreturnableStatements.has(lastNode.type)) return
+        try {
+          if (unreturnableStatements.has(lastNode.type)) return
```

Comment:
The obvious perceivable error here would be something along the lines of Cannot read property 'type' of null or undefined, so is a try/catch and using errors for control flow needed here?